microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -392,6 +390,15 @@ def add_sample_apps_to_build_list(self, apps: Union[List[str], None]) -> None:
         else:
             self._sample_apps_to_build = []
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self._dpdk_source = kwargs.pop("dpdk_source", PACKAGE_MANAGER_SOURCE)
+        self._dpdk_branch = kwargs.pop("dpdk_branch")
```

Comment:
write it like `kwargs.pop("dpdk_branch", "")`, give it a default value.

Reply:
update

Reply:
is it pushed?

Reply:
by default, dpdk_source is given PACKAGE_MANAGER_SOURCE, dpdk_branch is not used if install dpdk from package manager

Reply:
It's about argument design. Because it's not used, the dpdk_branch is able to be not passed when calling it. But if there is no default value given, there will be an error. So the default value is to make the argument is optional.

Reply:
update :)